src/react-components/in-world-chat-box.js

```diff
@@ -131,11 +131,8 @@ class InWorldChatBox extends Component {
               styles.messageEntryButtonInRoom,
               styles.messageEntrySubmit
             ])}
-          >
-            <i>
-              <FontAwesomeIcon icon={faPaperPlane} />
-            </i>
-          </button>
+            dangerouslySetInnerHTML={{ __html: sendMessageIcon }}
```

Comment:
If we start using this method more often for icons, it would be nice to wrap it in a small react component so that we don't litter `dangerouslySetInnerHTML` all over the code base.